packages/documentation/src/components/pageFooter/Documentation.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Link } from 'gatsby';
+
+import sidebarData from '../../sidebar';
+
+const getIndexMdx = (location) => {
+  const sidebarSection = sidebarData.sections.find((section) =>
+    location.pathname.includes(section.href),
+  );
+
+  // Test first Level of the sidebar object
+  const level1 = sidebarData.sections.find(
+    (section) => section.href === location.pathname,
+  );
+
+  if (level1 && level1.indexmdx) {
+    return '/index';
+  }
+
+  // Test second Level of the sidebar object
+  if (sidebarSection?.items) {
+    const level2Sections = sidebarSection.items.map((item) => {
+      return item.items
+        ? item.items.find((lvl2Item) => lvl2Item.href === location.pathname)
+        : null;
+    });
+
+    const level2 = level2Sections.map((section) => {
+      if (section && section.href && section.href === location.pathname) {
+        return section && section.indexmdx ? true : false;
+      }
+    });
+
+    if (level2.find((indexmdx) => indexmdx)) {
```

Comment:
I'm not following this 100%. Is the goal to find the first truthy value in `level2`, or is the goal to find the first object where `{ indexmdx: true }`?

Reply:
Correct!
The goal to find the first object where `{ indexmdx: true }`